tools/DfmHttpService/DfmHttpServer.cs

```diff
@@ -35,6 +35,7 @@ public void Start()
 
             _listener.Start();
             _processing.Reset();
+            Console.WriteLine("Ready");
```

Comment:
got it, how about some comments here?

---
In reply to: [108114901](https://github.com/dotnet/docfx/pull/1490#discussion_r108114901) [](ancestors = 108114901,108107907)